src/linter.ts

```diff
@@ -2,7 +2,11 @@ import { createFormat, FormatFunction } from "./format";
 import { createLint, LintFunction } from "./lint";
 import { LinterAdapter } from "./linter-adapter";
 import { loadLinterProvidersFromFile } from "./linter-provider";
-import { getDefaultLogLevel, logger } from "./logger";
+import { logger } from "./logger";
+
+export interface LinterOptions {
+  logLevel?: "error" | "warn" | "info" | "debug";
```

Comment:
According to this `winston` uses the `npm` log level which includes the following.
```
{ 
  error: 0, 
  warn: 1, 
  info: 2, 
  http: 3,
  verbose: 4, 
  debug: 5, 
  silly: 6 
}
```

Maybe the change [here](https://github.com/linterjs/core/pull/32/files#diff-67839461ab481509d67e9ff075da7000R62) should be `silly` instead of debug?

Reply:
`winston` itself can be configured

We could dicide to use npm's log levels or the one from syslog or even our own set

Only "problem" is, that there are only some methods like `logger.debug`
To log something like `trace`, we need to use `logger.log('trace', 'message')` _I don't like that_
So maybe we should use our own set and only use levels that can be called directly

... method for `logger.silly('message')` exists too, but not `trace`
So yeah maybe we should use npm's defaul set

Reply:
But I will remove `'http'`, cause we don't need this